src/main/scala/scorex/core/network/NodeViewSynchronizer.scala

```diff
@@ -57,18 +56,19 @@ PMOD <: PersistentNodeViewModifier, HR <: HistoryReader[PMOD, SI] : ClassTag, MR
 
   protected val deliveryTimeout: FiniteDuration = networkSettings.deliveryTimeout
   protected val maxDeliveryChecks: Int = networkSettings.maxDeliveryChecks
+  protected val maxRequestedPerPeer: Int = networkSettings.maxRequestedPerPeer
   protected val invSpec = new InvSpec(networkSettings.maxInvObjects)
   protected val requestModifierSpec = new RequestModifierSpec(networkSettings.maxInvObjects)
   protected val modifiersSpec = new ModifiersSpec(networkSettings.maxPacketSize)
 
   protected val msgHandlers: PartialFunction[(MessageSpec[_], _, ConnectedPeer), Unit] = {
-    case (_: SIS @unchecked, data: SI @unchecked, remote) => processSync(data, remote)
-    case (_: InvSpec, data: InvData, remote)              => processInv(data, remote)
-    case (_: RequestModifierSpec, data: InvData, remote)  => modifiersReq(data, remote)
-    case (_: ModifiersSpec, data: ModifiersData, remote)  => modifiersFromRemote(data, remote)
+    case (_: SIS@unchecked, data: SI@unchecked, remote) => processSync(data, remote)
+    case (_: InvSpec, data: InvData, remote) => processInv(data, remote)
+    case (_: RequestModifierSpec, data: InvData, remote) => modifiersReq(data, remote)
+    case (_: ModifiersSpec, data: ModifiersData, remote) => modifiersFromRemote(data, remote)
   }
 
-  protected val deliveryTracker = new DeliveryTracker(context.system, deliveryTimeout, maxDeliveryChecks, self)
+  protected val deliveryTracker = new DeliveryTracker(context.system, deliveryTimeout, maxDeliveryChecks, maxRequestedPerPeer, self)
```

Comment:
Consider also the existing changes in SDK: `SidechainDeliveryTracker` class. And it's usage at `SidechainNodeViewSynchronizer`.
Probably, we can do some additional improvements here to simplify the SDK integration.

Reply:
There's a single method added in SDK.
Do you want to move it to scorex?
![image](https://user-images.githubusercontent.com/64777406/182146973-ed4b45d0-62b6-464f-85d2-19e7efb88fa4.png)

Reply:
done